larpandoracontent/LArObjects/LArMCParticle.h

```diff
@@ -103,6 +121,15 @@ inline LArMCParticle::LArMCParticle(const LArMCParticleParameters &parameters) :
     object_creation::MCParticle::Object(parameters),
     m_nuanceCode(parameters.m_nuanceCode.Get())
 {
+    for (auto const &mcStepPosition : parameters.m_mcStepPositions)
+    {
```

Comment:
Because there's only one nested line in each of the two for loops, you can probably drop the braces